SourceCode/AgOpenGPS.Core/Presenters/FieldStreamerPresenter.cs

```diff
@@ -0,0 +1,76 @@
+﻿﻿using AgOpenGPS.Core.Interfaces;
+using System;
+
+namespace AgOpenGPS.Core.Presenters
+{
+    public class FieldStreamerPresenter : IFieldStreamerPresenter
+    {
+        private readonly IErrorPresenter _errorPresenter;
+
+        public FieldStreamerPresenter(IErrorPresenter errorPresenter)
+        {
+            _errorPresenter = errorPresenter;
+        }
+
+        void IFieldStreamerPresenter.PresentBoundaryFileMissing()
+        {
+            ShowStreamingError("gStr.gsMissingBoundaryFile");
```

Comment:
Shouldn't you be using the translated text, instead of its string representation?
```suggestion
            ShowStreamingError(gStr.gsMissingBoundaryFile);
```